### DISCUSSION TITLE
Public e-mail isn't shown

### DISCUSSION THREAD

**Discussion by Wdestroier** [community/community#48609] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

Hey,

I have noticed my public profile only displays my verified, non-primary, public e-mail address to myself.

Account settings:
![image](https://user-images.githubusercontent.com/14764307/221385383-f18f021b-8dc6-48b9-964d-f4ecfd2f34a7.png)
Public profile when not logged-in:
![image](https://user-images.githubusercontent.com/14764307/221385399-e1b4060f-c99b-42e2-a1c1-0f5d6b06ae05.png)
Public profile when logged-in:
![image](https://user-images.githubusercontent.com/14764307/221385531-fa18c5ca-f9e9-4533-820b-5a40f3e6991a.png)

**Comment by valentinojosh** [comment]
Hi!
It is possible your email address is set to private. 

- First, make sure you verified you email
- Go to email settings on GitHub: https://github.com/settings/emails
- Make sure "Keep my email addresses private" is NOT ticked
- Next go to Public Profile in settings: https://github.com/settings/profile
- Make sure public email is selected on the desired email

That should cover most of the possible issues. Hope this helps!

**Comment by Wdestroier** [comment]
I have double checked if the e-mail was verified:
![image](https://user-images.githubusercontent.com/14764307/221386467-d5aa6bf7-5619-437b-8226-045b9cfbd13a.png)

Unticked "Keep my email addresses private" (I prefer to keep all e-mails private except one, but it's fine):
![image](https://user-images.githubusercontent.com/14764307/221386469-81c13215-dcdd-435a-8a14-cec7ea3ab635.png)

Set the public e-mail again:
![image](https://user-images.githubusercontent.com/14764307/221386481-f5d47f8b-f168-4363-8b31-15d8c3d17cb2.png)

I still can't see the e-mail when I'm not logged-in 😕

**Reply by valentinojosh** [reply]
Hi again,
Seems the issue is probably that you're viewing without being logged in.

![image](https://user-images.githubusercontent.com/53629621/221388298-e9eff761-79c7-45c9-bcf2-0ca013d84e9d.png)

Im able to see your email!

**Comment by melvincwng** [comment, marked as answer]
Hey @Wdestroier, email addresses are **only** visible to GitHub users who are logged in.

If you are not logged in to GitHub, you won't be able to see the email address. My guess is that this is probably done to prevent bots from scraping and obtaining your email address.

If you need additional reference, I have previously answered a similar question in #28575.

Hope this answers your question 😃

Take care and have a good day 👋

**Comment by Wdestroier** [comment]
Thank you very much @valentinojosh and @melvincwng!
